BeerDispancer/BeerDispencer.Infrastructure/Persistence/BeerDispencerUof.cs

```diff
@@ -65,6 +93,8 @@ public void Dispose()
             _transaction?.Dispose();
```

Comment:
Why not in dispose method?

Reply:
code is looking like that: separate methods for COmmit and Dispose

  public void CommitTransaction()
        {
            _transaction.Complete();
        }

        public void Dispose()
        {
            _transaction?.Dispose();
            _dbcontext?.Dispose();
        }

Reply:
Mixed up) Sorry)